plugins/SegmentEditor/lang/en.json

```diff
@@ -36,6 +36,21 @@
         "SegmentXIsAUnionOf": "%s is a union of these segments:",
         "CustomSegment": "Custom Segment",
         "SegmentOperatorIsNullOrEmpty": "is null or empty",
-        "SegmentOperatorIsNotNullNorEmpty": "is not null nor empty"
+        "SegmentOperatorIsNotNullNorEmpty": "is not null nor empty",
+        "UnprocessedSegmentNoData1": "These reports have no data, because the Segment you requested %1$s has not yet been processed by the system.",
+        "UnprocessedSegmentNoData2": "Data for this Segment will become available in a few hours when processing completes.",
+        "UnprocessedSegmentInVisitorLog1": "%1$sMeanwhile you can use the Visitor Log%2$s to test whether your segment will match your users correctly by applying it there.",
+        "UnprocessedSegmentInVisitorLog2": "When applied, you can see immediately which visits and actions were matched by your segment.",
+        "UnprocessedSegmentInVisitorLog3": "This can help you confirm your Segment matches the users and actions you expect it to.",
+        "UnprocessedSegmentApiError1": "The Segment '%s' is set to '%s' but Matomo is not currently configured to process segmented reports in API requests.",
+        "UnprocessedSegmentApiError2": "To see data for this report in the future, you will need to edit your segment and choose the option labeled '%s'.",
+        "UnprocessedSegmentApiError3": "Then after a few hours your segment data will become available through the API.",
+        "CustomUnprocessedSegmentApiError1": "The Segment you requested has not yet been created in the Segment Editor and so the report data has not been pre-processed.",
+        "CustomUnprocessedSegmentApiError2": "To see data for this segment, you must go to Matomo and create this segment manually in the Segment Editor.",
+        "CustomUnprocessedSegmentApiError3": "(Alternatively, you can create a new segment programatically using the SegmentEditor.add API method).",
+        "CustomUnprocessedSegmentApiError4": "Once created the segment in the editor (or via API), this error message will disappear and within a few hours you will see your segmented report data, after the segment data has been pre-processed.",
+        "CustomUnprocessedSegmentApiError5": "Please note that you can test whether your segment will work without having to wait for it to be processed by using the Live.getLastVisitsDetails API.",
+        "CustomUnprocessedSegmentApiError6": "When using this API method, you will see which users and actions were matched by your &segment= parameter.",
+        "CustomUnprocessedSegmentNoData2": "To see data for this segment, you must create this segment manually in the Segment Editor, then wait a couple hours for preprocessing to complete."
```

Comment:
maybe this could be renamed to `CustomUnprocessedSegmentNoData` as there was no other key called this?